src/server/api/controllers/cvs.controller.js

```diff
@@ -48,10 +42,8 @@ const editCv = async (cvId, updatedCv) => {
     });
 };
 
-const deleteCv = async (cvId) => {
-  return knex('cv')
-    .where({ id: cvId })
-    .del();
+const deleteCv = async (cvsId) => {
+  return knex('cvs').where({ id: cvsId }).del();
 };
```

Comment:
good job charles. this is a clean code!! i have tested and everything works good. cheers!

Reply:
Thanks Diego,.,. cheers